### PR TITLE
Submitting Assessment

### PR DESCRIPTION
**Pull request by Kensuga**

### CODE REVIEW

File: code-challenges.js [hunk 1]
```diff
@@ -2,35 +2,43 @@
...
 
-
-
-
-
 var hand1 = [5, 5, 5, 3, 3]
 // Expected output: true
 var hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
+
```
Comment (sjproctor) [inline code-challenges.js:37]: This does work for the test variables, but it wouldn't work for all test cases.

File: rails-questions.md [hunk 2]
```diff
@@ -2,26 +2,33 @@
...
 
 2. Explain each section of the following route:  get '/library/:id' => 'book#show'
 
-get -
-library -
-:id -
-book -
-show -
+get - Get grabs the address in the addressbar and runs the method associated
+library - library is a referance to a view
```
Comment (sjproctor) [inline rails-questions.md:10]: library is the url path

**Review by sjproctor — APPROVED** [review]
Great job! You are the regex king.

File: code-challenges.js (hunk 1 above)
Comment (Kensuga) [reply code-challenges.js:37]: You are right, I forgot to update the code, but I rewrote it to work for all cases.

function fullHouse(arr) {
  return Object.values(arr.reduce((p,n)=>(p[n]=1+(p[n]||0),p),{})).sort((a,b)=>a-b)+""=="2,3"
}

**Pull request merged**